scripts/cnn_variant_wdl/README.md

```diff
@@ -0,0 +1,68 @@
+# gatk4-cnn-variant-filter
+
+### Purpose :
+This repo provides workflows that takes advantage of GATKs CNN tool which is a deep learning 
```

Comment:
takes -> take
GATKs -> GATK's
Maybe instead of "This repo", "This directory"?  Or "Here are workflows that..."
comma after "which"